extensions/wikia/Flags/controllers/FlagsController.class.php

```diff
@@ -45,21 +45,88 @@ private function processRequest() {
 	 */
 
 	/**
-	 * Retrieves all data for flags assigned to the given page.
+	 * Retrieves all data for all flag types available on the given wikia
+	 * with an intent of rendering a modal with an edit form.
+	 * It returns
+	 * @see getFlagsForPageForEdit()
```

Comment:
Oops, you've found an unfinished part. I've corrected it and will push it with the next PR.